modules/installation-requirements-user-infra.adoc

```diff
@@ -197,6 +197,77 @@ On your z/VM instances, set up:
 * 16 GB for the temporary {product-title} bootstrap machine
 endif::ibm-z[]
 
+ifdef::ibm-power[]
+[id="minimum-ibm-power-system-requirements_{context}"]
+== Minimum IBM Power Systems requirements
+
+You can install {product-title} version {product-version} on the following IBM hardware:
+
+* IBM Power8 or Power9
```

Comment:
IBM POWER8 or POWER9 processor-based systems 

@lsmcfadden Looking at the Power Guidelines (page 4) I think it should be written like this. However, I'm not the Power expert please confirm.